lib/Dialect/FIRRTL/Transforms/LowerLayers.cpp

```diff
@@ -138,7 +138,7 @@ class LowerLayersPass
     : public circt::firrtl::impl::LowerLayersBase<LowerLayersPass> {
   hw::OutputFileAttr getOutputFile(SymbolRefAttr layerName) {
     auto layer =
-        cast<LayerOp>(SymbolTable::lookupSymbolIn(getOperation(), layerName));
+        cast<LayerOp>(symbolTable->lookupSymbolIn(getOperation(), layerName));
```

Comment:
I think you might have to call a different function, shouldn't have to take the symbol operation as an argument

Reply:
So, this may not work so easily then. As this is a `SymbolRefAttr` and it needs to do a nested lookup across symbol tables. It may be better to instead do the actual `SymbolRefAttr` -> `LayerOp` caching during the serial portion, then.

WDYT?

Reply:
Yeah, that sounds like a solid plan.  I don't see a good way to force the SymbolTableCollection to prepopulate the symbol tables.

Reply:
Fixed in the force push here: https://github.com/llvm/circt/pull/7436/commits/46326f7aef18f4ce7ebaf5de39f314c37e0a8379